src/plugins/data_source_management/public/components/direct_query_data_sources_components/associated_object_management/associated_objects_details_flyout.test.tsx

```diff
@@ -106,13 +106,9 @@ describe('AssociatedObjectsDetailsFlyout', () => {
     renderComponent();
     fireEvent.click(screen.getAllByRole('button')[0]);
     await waitFor(() => {
-      expect(mockApplication.navigateToApp).toHaveBeenCalledWith('observability-logs', {
-        path: '#/explorer',
-        state: {
-          datasourceName: 'testDatasource',
-          datasourceType: 's3glue',
-          queryToRun: 'source = testDatasource.testDatabase.testTable | head 10',
-        },
+      expect(mockApplication.navigateToApp).toHaveBeenCalledWith('data-explorer', {
```

Comment:
nit: could use the util functions you made here in the tests too right?

Reply:
Yeah I think the previous tests that existed were just there for coverage purposes, I don't think I should call on the util functions since I believe that's what the tests are testing here.